lib/pbench/server/api/resources/__init__.py

```diff
@@ -28,7 +28,7 @@
     MetadataBadKey,
     MetadataError,
 )
-from pbench.server.database.models.server_config import ServerConfig
+from pbench.server.database.models.server_configs import ServerConfig
```

Comment:
`server_configs` strikes me as a mismatch for `ServerConfig`, and, as I said elsewhere, I think it should be singular.

Reply:
RE: singular. Yes, maybe. While I don't necessarily mind matching the module name to the *table* name, as Peter seems to want to do, the table is more or less incidental. What the module does is define the contents and behavior of a single *instance*, a *row* of the table: the fact that it's implemented on top of SQLAlchemy and declares a SQL table name is rarely that important outside the module.

So if `dataset.py` defines the `Dataset` class which maps to the `datasets` SQL table, I'd be happy. (And, yes "metadata" is an exception because the "metadata" name is reserved, which is why I used "dataset_metadata" from the beginning, which remains "correct" unless we choose a singular table name convention and really want to deal with "dataset_metadatum", which I very definitely don't!)

We've been inconsistent about this, and I guess by "we" I have to single out Nikhil and me. If someone is going to bother to "clean it up" (which I've almost done a million times) I'd prefer to see the model I laid out above, with class and module singular, and table plural. And if that mismatch in case really bugs someone, I'll go with a singular table name too: I like the "feel" of a SQL statement better with the plural, but we rarely have to look at or directly generate SQL so it's not a big deal.

Reply:
Please see how `ServerSetting` and `server_settings` looks and feels.